plugins/gui/src/TransferFunctionEditor.cpp

```diff
@@ -133,8 +139,9 @@ void RainbowAdapter(param::TransferFunctionParam::TFNodeType& nodes, size_t n) {
     }
 }
 
-std::array<std::tuple<std::string, PresetGenerator>, 12> PRESETS = {
-    std::make_tuple("Select...", [](auto& nodes, auto n) {}), std::make_tuple("Ramp", RampAdapter),
+std::array<std::tuple<std::string, PresetGenerator>, 20> PRESETS = {
```

Comment:
Now that we have more and more color maps, this tuple could/should be split into a struct, e.g..:
```
struct Preset {
	std::string name;
	enum class Scale {
		Sequential,
		Diverging,
		Qualitative
	} scale;
	std::string notice; // harmful, de-pinked, ...
	std::string reference; // ColorBrewer, Paper, ...
};
```
(just an idea)